pegasus/helpers/section_api_helpers.rb

```diff
@@ -443,10 +445,24 @@ def students
           {
             location: "/v2/users/#{row[:id]}",
             age: DashboardStudent.birthday_to_age(row[:birthday]),
-            completed_levels_count: DashboardStudent.completed_levels(row[:id]).count
           }
         )
       end
+    # Though it would be simpler to query the level counts for each student via
+    # DashboardStudent#completed_levels and inject them to @students via the row.merge above,
+    # querying all students together (as below) is significantly more performant.
```

Comment:
If/when we move these sorts of queries to dashboard, where we use ActiveRecord instead of Sequel, are we going to lose the benefits of all these sorts of performance optimizations, or will porting them being fairly simple?